package.json

```diff
@@ -50,4 +50,4 @@
     "request": "~2.34.0",
```

Comment:
Did the Snyk updater not run on dev dependencies? Request is the one we're getting alerted about.